bites_test.py

```diff
@@ -116,9 +154,447 @@ def test_can_initialise_anthill_as_list_with_len_five_and_every_element_is_None(
+    self.assertEqual(fake_luigi.take_turn.call_count, 2)
+    self.assertEqual(fake_mario.take_turn.call_args_list[0], mock.call(
+      starting_trail, starting_ant_positions, starting_anthill))
+    self.assertEqual(fake_luigi.take_turn.call_args_list[0], mock.call(
+      trail_after_turn_1_mario, ant_pos_after_turn_1_mario, anthill_after_turn_1_mario))
+    self.assertEqual(fake_mario.take_turn.call_args_list[1], mock.call(
+      trail_after_turn_2_luigi, ant_pos_after_turn_2_luigi, anthill_after_turn_2_luigi))
+    self.assertEqual(fake_luigi.take_turn.call_args_list[1], mock.call(
+      trail_after_turn_3_mario, ant_pos_after_turn_3_mario, anthill_after_turn_3_mario))
+
+  def test_the_game_is_played_until_all_ants_are_on_the_anthill(self):
+  # test 78
+    """
+    The final move of a game;
+    P0 moves purple ant to anthill[0]
+    """
+    starting_trail = [
+      "apple",
+      None,
+      None,
+      "pepper",
+      None,
+      None,
+      None,
+      "bread",
+      "pepper",
+      None]
+    starting_ant_positions = {
+      "purple": None,
+      "brown": "anthill",
+      "green": "anthill",
+      "red": "anthill",
+      "yellow": "anthill"}
+    starting_anthill = [None, "brown", "green", "red", "yellow"]
+
+    trail_after_turn_1_mario = starting_trail
+    ant_pos_after_turn_1_mario = {
+      "purple": "anthill",
+      "brown": "anthill",
+      "green": "anthill",
+      "red": "anthill",
+      "yellow": "anthill"}
+    anthill_after_turn_1_mario = ["purple", "brown", "green", "red", "yellow"]
+
+    expected_new_trail = trail_after_turn_1_mario
+    expected_new_ant_positions = ant_pos_after_turn_1_mario
+    expected_new_anthill = anthill_after_turn_1_mario
+
+    ants = ["red", "yellow", "green", "brown", "purple"]
+    tokens_for_trail = {}
+    
+    fake_mario = mock.MagicMock()
+    fake_mario.take_turn = mock.MagicMock(return_value = (
+      trail_after_turn_1_mario, ant_pos_after_turn_1_mario, anthill_after_turn_1_mario))
+
+    players = [fake_mario]
+    bites_game = Bites(ants, tokens_for_trail, players)
+    bites_game.trail = starting_trail
+    bites_game.ant_positions = starting_ant_positions
+    bites_game.anthill = starting_anthill
+    bites_game.take_all_turns()
+
+    self.assertEqual(fake_mario.take_turn.call_count, 1)
+    self.assertEqual(fake_mario.take_turn.call_args_list[0], mock.call(
+      starting_trail, starting_ant_positions, starting_anthill))
+    self.assertEqual(bites_game.trail, expected_new_trail)
+    self.assertEqual(bites_game.ant_positions, expected_new_ant_positions)
+    self.assertEqual(bites_game.anthill, expected_new_anthill)
+
+  def test_final_scores_are_printed_at_the_end_of_the_game(self):
+  # test 79
+    """
+    Starting point is a game situation at the point that the penultimate turn has 
+    been completed. The final move will be executed here and the scores analysed.
+    P0 moves purple ant to anthill[0]
+    """
+    pass
```

Comment:
Why is this test all commented out?

Reply:
It was my mistake to include this as part of the pull-request. I drafted this test but then realised I would need the `print_scores()` method to be functional before this test would be able to pass. 
This test will now be moved to a new test class for the `play_full_game()` method which will be developed next.